README.md

```diff
@@ -23,17 +23,20 @@ Shotover currently supports intercepting requests for the following technologies
 
 * Cassandra (CQL4)
 * Redis (RESP2)
+* Kafka (Kafka Wire Protocol)
 
 It currently supports writing output to the following technologies (sinks):
 
 * Cassandra
 * Redis
+* Kafka
 
 ## What problems does Shotover solve?
 
 Concrete examples where Shotover has been applied include:
 
-* [Multi-region, active-active redis](shotover-proxy/tests/test-configs/redis-multi/)
+* [Multi-region, active-active redis](shotover-proxy/tests/test-configs/redis/multi)
+* [Multi-region Kafka](shotover-proxy/tests/test-configs/kafka/cluster/)
```

Comment:
We dont actually deploy shotover kafka anywhere, which is the point of this section, so lets not mention it here.

Reply:
Ah ok gotcha. I'll nuke this bit.